### PR TITLE
adding-wrap-doc-method-16

### PR DESCRIPTION
**Pull request by rahul799**
# Description

Please include a summary of the change and which issue is fixed.

# Checklist:
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

# Note:
- Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: src/pyOpenMS/pxds/TargetedExperimentHelper.pxd [hunk 1]
```diff
@@ -90,13 +90,13 @@ cdef extern from "<OpenMS/ANALYSIS/TARGETED/TargetedExperimentHelper.h>" namespa
...
         libcpp_vector[ RetentionTime ] rts
 
-        void setChargeState(int charge) nogil except +
-        int getChargeState() nogil except +
-        bool hasCharge() nogil except +
-        double getRetentionTime() nogil except +
-        bool hasRetentionTime() nogil except +
-        RTType getRetentionTimeType() nogil except +
-        RTUnit getRetentionTimeUnit() nogil except +
+        void setChargeState(int charge) nogil except + # wrap-doc:Set the peptide or compound charge state
```
[the remark this answers is not in the document]
Comment (rahul799) [reply src/pyOpenMS/pxds/TargetedExperimentHelper.pxd]: Hey @oliveralka 
Thank you for the review.
I have made the changes needed in this file.
I think there are many other instances where I will have to change. So I will raise a separate PR for that once all of my #wrap-doc PR's are merged. What do you think?

[the remark this answers is not in the document]
Comment (oliveralka) [reply src/pyOpenMS/pxds/TargetedExperimentHelper.pxd]: @rahul799 Yes, that sound great - thanks! :)

**Comment by oliveralka** [comment]
rebuild jenkins

**Review by oliveralka — APPROVED** [review]
Looks good - thanks a lot!

**Pull request merged**